templates/navigation-mobile-menu-items.twig

```diff
@@ -0,0 +1,63 @@
+{% for key,item in menu %}
+	{% set targetId = item.ID %}
+	{% if fn('get_post_meta', item.ID, '_menu_item_object_id', true ) == act_page_id %}
+		{% set link_ga_action = 'Act' %}
+	{% elseif fn('get_post_meta', item.ID, '_menu_item_object_id', true ) == explore_page_id %}
+		{% set link_ga_action = 'Explore' %}
+	{% else %}
+		{% set link_ga_action = item.title %}
+	{% endif %}
+	<li class="navigation-mobile-menu-item accordion-item {{ item.class }} {{ item == item.current ? 'active' : '' }}">
```

Comment:
I think `item.current` is a _bool_, so just `{{item.current ? 'active' : ''}}` should work